pkg/data/queue.go

```diff
@@ -0,0 +1,72 @@
+package data
+
+import (
+	"fmt"
+)
+
+// ErrorPlayerNotFound : Player specific errors
+var ErrorPlayerNotFound = fmt.Errorf("Player not found")
+
+// Player defines the structure for an player in queue.
+// Formatting done with json tags to the right. "-" : don't include when encoding to json
+type Player struct {
+	UserID          int     `json:"userid" validate:"required,notinqueue,exist"`
```

Comment:
I thought we wanted UserIP in here as well so that dispatcher doesn't have to keep the past requests in memory